tests/model_monitoring/test_application_steps.py

```diff
@@ -40,13 +40,15 @@
 
 class TestEventPreparation:
     ENDPOINT_ID = "test-ep-id"
+    ENDPOINT_NAME = "test-ep-name"
     APPLICATION_NAME = "test-app"
 
     @classmethod
     @pytest.fixture
     def controller_event(cls) -> dict[str, typing.Any]:
         return {
             mm_constants.ApplicationEvent.ENDPOINT_ID: cls.ENDPOINT_ID,
+            mm_constants.ApplicationEvent.ENDPOINT_NAME: cls.ENDPOINT_NAME,
```

Comment:
you can add the endpoint_name to the artifact labels assertion

Reply:
Done